### PR TITLE
add notes about using and configuring SwitchBot entities

### LINKED ISSUE #31550: Info on how to use the integration after HA detects devices?
### Feedback

This documentation page doesn't provide any info on how to actually use the integration, once it's picked up the device.

For example, I installed the switchbot bluetooth integration and it picked up my curtain 3 device. Now I have no idea what to do. How do I tell HA what position is "open"? How do I set it to use 2 bots to open/close both sides of my curtains? What is "position mode" (shows up in the UI, but no idea what it is)?

These integration features don't seem to be mentioned in the docs and it's not clear how to set them up or use them.

### URL

https://www.home-assistant.io/integrations/switchbot/

### Version

2024.2.2

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by jtaseff**
## Proposed change
Adding a paragraph to clarify some general details about actually using switchbot entities, since each of them shows up under its own domain and there is not any documentation on this page about the triggers and services available for each of those domain entities. 
Plus a note that some switchbot devices have to be set up within the app first.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [X] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
- This PR fixes or closes issue: fixes #31550

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/switchbot.markdown [hunk 1]
```diff
@@ -47,6 +47,10 @@ Please note, device names configured in the SwitchBot app are not transferred in
 
 {% include integrations/config_flow.md %}
 
+SwitchBot devices are added to Home Assistant according to their domain, such as Curtains under the cover domain. For more information how to use these in automations, see the buliding block integration that domain, such as [cover](/integrations/cover/).
```
Comment (frenck) [inline source/_integrations/switchbot.markdown]: This sentence feels odd and more like a base concept and not related to SwitchBot at all. IMHO, adding this isn't right, as it causes this to be added to each entity platform we have for each integration page.

File: source/_integrations/switchbot.markdown [hunk 2]
```diff
@@ -47,6 +47,10 @@ Please note, device names configured in the SwitchBot app are not transferred in
...
+Some SwitchBot devices need to be configured within the app before controlling them from Home Assistant, such as adjusting the cover open/close limits or pairing two covers to move together.
```
Comment (frenck) [inline source/_integrations/switchbot.markdown]: This is a prerequisite which should be documented before the config flow section.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/switchbot.markdown (hunk 1 above)
Comment (jtaseff) [reply source/_integrations/switchbot.markdown]: I'm curious what you think of this other PR #31789 I made to do address this idea in a more general/automated way. I just feel it's unclear for newer users if they end up on any specific integration page like Switchbot, there is no mention of how to automate with the entities it made.
Thanks for the feedback!

File: source/_integrations/switchbot.markdown [hunk 3]
```diff
@@ -45,8 +45,13 @@ If you have multiple devices of the same type, you need to get the BTLE MAC addr
 
 Please note, device names configured in the SwitchBot app are not transferred into Home Assistant.
 
+Some SwitchBot devices need to be configured within the app before controlling them from Home Assistant, such as calibrating the cover open/close limits or pairing two covers to move together.
```
Comment (frenck) [inline source/_integrations/switchbot.markdown]:
```suggestion
Some SwitchBot devices need to be configured within the app before being controlled by Home Assistant, such as calibrating the cover open/close limits or pairing two covers to move together.
```

File: source/_integrations/switchbot.markdown [hunk 4]
```diff
@@ -45,8 +45,13 @@ If you have multiple devices of the same type, you need to get the BTLE MAC addr
 
 Please note, device names configured in the SwitchBot app are not transferred into Home Assistant.
 
+Some SwitchBot devices need to be configured within the app before being controlled by Home Assistant, such as calibrating the cover open/close limits or pairing two covers to move together.
+
 {% include integrations/config_flow.md %}
 
+
+
+
```
Comment (frenck) [inline source/_integrations/switchbot.markdown]:
```suggestion
```

**Review by frenck — APPROVED** [review]
Thanks, @jtaseff 👍 

../Frenck

**Pull request merged**